packages/webapp/src/webview/ui/components/FeedbackInterface/FeedbackInterface.scss

```diff
@@ -0,0 +1,63 @@
+.feedback-subcontainer {
+    display: flex;
+    justify-content: flex-start;
+    gap: 20px;
+    margin-top: 15px;
+}
+
+.feedback-container {
+    h3 {
+        color: var(--vscode-settings-headerForeground);
+        font-size: 14px;
+        font-weight: 700;
+        margin-bottom: 0px;
+    }
+
+    margin-bottom: 15px;
+}
+
+.solved-box {
```

Comment:
There is css rules duplication in solved-box and not-solved-box classes. Can we have common css rules instead, please?